models/tokens/ethereum/tokens_ethereum_transfers.sql

```diff
@@ -0,0 +1,17 @@
+{{ config(
+        alias = alias('transfers'),
+        tags=['dunesql'],
+        materialized = 'view',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "sector",
+                                    "fungible",
+                                    \'["hildobby", "aalan3"]\') }}'
+        )
+}}
+
+
+{{transfers_enrich(
+    blockchain='ethereum',
+    transfers_base = ref('tokens_ethereum_transfers_base'),
+    tokens_erc20 = ref('tokens_erc20'),
```

Comment:
this should be static across chains, so we can remove it from the variables and hardcode the ref in the macro?